governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md

```diff
@@ -0,0 +1,61 @@
+---
+title: JAT1 to JAT2 Asset Reallocation - September 4, 2023
+summary: Signal your support or opposition for reallocating USD 500M from JAT1's ETF holdings to the T-bills ladder under JAT2.
+discussion_link: https://forum.makerdao.com/t/proposal-to-revise-asset-allocation-of-jat1-and-jat2/21718
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-09-04T16:00:00
+end_date: 2023-09-07T16:00:00
+---
+# Poll: JAT1 to JAT2 Asset Reallocation - September 4, 2023
+
+The Reserve Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 4 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+* Selling the ETF products within JAT1.
+* Paying any proceeds from this sale after expenses in excess of 500 million USD into the surplus buffer.
```

Comment:
Link to Surplus Buffer Documentation:

https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer